### PR TITLE
add video-wrapup of dconf

### PR DESCRIPTION
**Pull request by wilzbach**
From Andrei:
http://forum.dlang.org/post/ni2flg$2ge3$1@digitalmars.com

CC @mdparker - we should probably keep an eye on this news spot and update it from time to time.
Now that we have @CyberShadow's awesome auto-deploy, there's no excuse ;-)

### CODE REVIEW

File: index.dd [hunk 1]
```diff
@@ -83,9 +83,9 @@ $(DIVID news,
 $(DIVC boxes,
     $(DIVC row,
         $(TOUR newspaper-o, News,
-            $(P The D foundation got accepted for the Google Summer of Code 2016
-                and $(LINK2 https://summerofcode.withgoogle.com/organizations/5078256051027968/, four projects)
-                will be pursued over the summer.
+            $(P $(LINK2 http://dconf.org/2016/index.html, DConf 2016) has concluded,
+                $(LINK2 https://vimeo.com/167235872, take a look),
+                and see you next year.
```
Comment (CyberShadow) [inline index.dd]: This sounds a bit odd. Other text proposals?

Comment (wilzbach) [inline index.dd]: - DConf 2016 was a big and bright week - we are looking forward to seeing you next year
- DConf 2016 was a big and bright week, watch the summary and see you next time.
- DConf 2016 was a full success, take an impression on hopefully see you next year
....

Comment (wilzbach) [inline index.dd]: I would vote the first one "DConf 2016 was a big and bright week - we are looking forward to seeing you next year", other proposals?

**Comment by wilzbach** [comment]
> Let's go with that.

Rebased ;-)

(As said we should try to keep an eye on the news section and update it more regularly. )

**Pull request merged**